plugins/webp-uploads/helper.php

```diff
@@ -411,6 +411,17 @@ function webp_uploads_is_fallback_enabled(): bool {
 	return (bool) get_option( 'perflab_generate_webp_and_jpeg' );
 }
 
+/**
+ * Checks if the `perflab_generate_fallback_all_sizes` option is enabled.
+ *
+ * @since n.e.x.t
+ *
+ * @return bool True if the option is enabled, false otherwise.
+ */
+function webp_uploads_is_fallback_all_sizes_enabled(): bool {
```

Comment:
Maybe a better name would be something like `webp_uploads_should_generate_all_fallback_sizes()`?